gensim/test/test_word2vec.py

```diff
@@ -38,7 +38,7 @@ class LeeCorpus(object):
     def __iter__(self):
         with open(datapath('lee_background.cor')) as f:
```

Comment:
@tmylk Should be `smart_open` (we want to use `smart_open` consistently across gensim).